### DISCUSSION TITLE
Problems with dpgen init

### DISCUSSION THREAD

**Discussion by hxk0311** [deepmodeling/dpgen#1281] [Q&A]
When I run dpgen run on the server, I get the following error
![image](https://github.com/deepmodeling/dpgen/assets/139231442/7a82481e-e192-4dc0-b9ea-84038a90edd6)
![image](https://github.com/deepmodeling/dpgen/assets/139231442/76f9aa26-4ef3-4507-8ff6-5aa9a21467bd)

**Comment by njzjz** [comment]
The error message shows that the error occurred in the `fp` section, not the `train` section.
